demisto_sdk/commands/lint/resources/pylint_plugins/certified_partner_level_checker.py

```diff
@@ -142,6 +146,13 @@ def _return_outputs_checker(self, node):
         except Exception:
             pass
 
+    def _check_demisto_results(self, node):
+        try:
+            if node.func.attrname == "results" and node.func.expr.name == "demisto":
+                self.add_message("demisto-results-exists", node=node)
+        except Exception:
+            pass
```

Comment:
Can you please add a comment what're the expected exceptions and why you are ignoring them

Reply:
i did not add this exception, it was originally there,

Reply:
I see, it's still a bad practice. Make sure to at least add a log in case the exception is raised.